package.json

```diff
@@ -5,8 +5,8 @@
   "main": "static/canjs",
   "scripts": {
     "start": "node make-example.js -f",
-    "styles": "rm -rf node_modules/bit-docs-generate-html/site/static && node make-example.js -f",
-    "test": "npm run testee",
+    "styles": "rm -rf node_modules/bit-docs-generate-html/site/static && npm start",
+    "test": "npm start && npm run testee",
```

Comment:
We need to [generate the site before running the tests](https://github.com/canjs/bit-docs-html-canjs/commit/9b18d768ba11712c3779a6a81a74622637c5d863) so it always works.